There appears to be no necessary change in pkgconfig-1.5.1 needed
and we only ship 1.4.0 in pexos.
--- a/setup.py
+++ b/setup.py
@@ -390,7 +390,7 @@
 src_root = Path(__file__).parent / 'src'
 sources = [str(p.absolute()) for p in src_root.rglob('*.c')]
 pyxmlsec = Extension('xmlsec', sources=sources)
-setup_reqs = ['setuptools_scm>=3.2', 'pkgconfig>=1.5.1', 'lxml>=3.8']
+setup_reqs = ['setuptools_scm>=3.2', 'pkgconfig>=1.4.0', 'lxml>=3.8']
 
 
 with io.open('README.rst', encoding='utf-8') as f:
